examples/http2/index.js

```diff
@@ -0,0 +1,38 @@
+try {

```

Comment:
Remove this try wrapper. This is a http2 example, ideally the examples are how folks will write their code. Do you think everyone will be wrapping their servers in try blocks when they start using http2?

Reply:
Aren't the tests running the examples? Maybe I was under a false impression here

Reply:
Nm, my bad, removing :)